src/core.js

```diff
@@ -562,7 +562,7 @@ $.extend( $.validator, {
 			var lastActive = this.lastActive;
 			return lastActive && $.grep( this.errorList, function( n ) {
 				return n.element.name === lastActive.name;
-			} ).length === 1 && lastActive;
```

Comment:
hmm tests are failling now, so my change did break things.

I still dont get where `lastActive` is changed between line 563 and 565 so, I dont get why it is checked against to be "truethy" again...